tests/e2e/test_shlvl.py

```diff
@@ -0,0 +1,113 @@
+from conftest import PROMPT, get_command_output
+
+
+def check_shlvl_increment_recursively(shell_session, i, INITIAL_SHLVL):
```

Comment:
``` suggestion
def check_shlvl_increment_recursively(shell_session, i, initial_shlvl):
```
INITIAL_SHLVL is currently used as a variable, but its name suggests it should be a constant.
Ref (PEP 8): https://peps.python.org/pep-0008/#constants